### ISSUE TITLE
testing 

### ISSUE THREAD

**Issue by HermanSontrop** [ebailey78/shinyBS#20]
Hi Eric,

in my work I've extensively used shinyBS. We recently stopped using it due to the bootstrap 3 migration of Shiny by RStudio. If you need help testing the new bootstrap 3 version just let me know!

best Herman

**Comment by ebailey78** [comment]
Thanks Herman,

I actually submitted shinyBS to CRAN last Friday. But they had a couple of issues with it, primarily how I did the examples. I need to make some changes before I resubmit and I just haven't gotten the chance yet. The version in the "shinyBS3" branch should be ready to go, except for the examples. It is missing some of the features in the old version but much of that is because those features were removed from Bootstrap 3. If you want to install and test that branch I would appreciate it! My goal it to resubmit to CRAN this Friday.

`devtools::install_github("ebailey78/shinyBS", ref = "shinyBS3")`

Thanks again, and sorry that I haven't been able to get this done more quickly.
